build/tasks/promises_aplus_tests.js

```diff
@@ -1,17 +1,9 @@
-"use strict";
```

Comment:
Maybe let's rename JS files in `test/promises_aplus_adapters/` to have the `cjs` extension to align with `main`? They will work in Node as well.